apkid/apkid.py

```diff
@@ -130,7 +130,7 @@ def scan_apk(apk_path, rules, timeout, output_json):
     results = {}
     try:
         zf = zipfile.ZipFile(apk_path, 'r')
-        target_members = filter(lambda n: is_target_member(n), zf.namelist())
+        target_members = [n for n in zf.namelist() if is_target_member(n)]
```

Comment:
What different does this make? Why is it needed?

Reply:
Same as above. Not sure why this is better/needed.

Reply:
Just for your info -- timeit may help show the difference.
https://stackoverflow.com/questions/26540885/lambda-is-slower-than-function-call-in-python-why